capstone/src/main/webapp/club-registration.html

```diff
@@ -18,13 +18,17 @@ <h2>Club Registration</h2>
         <label for="name">Description:</label>
         <input type="text" name="description" id="club-description"/>
       </div>
+      <div class="form-group">
+        <label for="name">Website:</label>
+        <input type="text" name="website" id="club-website"/>
```

Comment:
I don't think you need a '/' at the end of input tags: https://www.w3schools.com/tags/tag_input.asp 
Please remove them for the rest of the file as well (unless they are needed for some reason?)

Reply:
Yeah idk why they were there